camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/SpecParameterUnwrappingTest.java

```diff
@@ -309,7 +298,7 @@ public void testCatalogConfigOverridesParameters() {
         CatalogItem<?, ?> item = catalog.getCatalogItem(SYMBOLIC_NAME, TEST_VERSION);
         AbstractBrooklynObjectSpec<?,?> spec = catalog.peekSpec(item);
         List<SpecParameter<?>> params = spec.getParameters();
-        assertEquals(params.size(), 3);
+        assertEquals(params.size(), NUM_ENTITY_DEFAULT_CONFIG_KEYS + 2);
```

Comment:
if we deployed i think it would be; here we're just doing `peekSpec` so it doesn't have the app config keys, it just has `NUM_ENTITY_DEFAULT_CONFIG_KEYS + ConfigEntityForTest.NUM_CONFIG_KEYS_DEFINED_HERE + NUM_CONFIG_KEYS_FROM_TEST_BLUEPRINT` = `1+1+1`